### PR TITLE
[4.0] contrast improved

### PR DESCRIPTION
**Pull request by puneeth-chanda**
Pull Request for Issue # 23863

### Summary of Changes
Contrast has been improved for field labels

### CODE REVIEW

File: build/media_source/system/scss/fields/joomla-field-switcher.scss [hunk 1]
```diff
@@ -130,7 +130,7 @@ joomla-field-switcher {
     span {
       position: absolute;
       top: 0;
-      color: $gray-600;
+      color: (--dark);
```
Comment (C-Lodder) [inline build/media_source/system/scss/fields/joomla-field-switcher.scss]: `var(--dark)`

**Review by C-Lodder — CHANGES_REQUESTED** [review]

File: build/media_source/system/scss/fields/joomla-field-switcher.scss (hunk 1 above)
Comment (puneeth-chanda) [reply build/media_source/system/scss/fields/joomla-field-switcher.scss]: sorry for this silly mistake

**Comment by puneeth-chanda** [comment]
@C-Lodder please review this

**Review by C-Lodder — APPROVED** [review]

**Comment by puneeth-chanda** [comment]
@wilsonge can you please review this.

**Comment by puneeth-chanda** [comment]
sorry by closed by mistake.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**